Jenkinsfile

```diff
@@ -95,19 +95,23 @@ pipeline {
             sh './bin/helm-dependency-update-in-docker'
           }
         }
-        stage('Openshift E2E Workflow Tests') {
-          steps {
-            sh 'cd bin/test-workflow && summon --environment openshift -D ENV=ci -D VER=current ./start --platform oc'
-          }
-        }
-        stage('Run E2E Tests') {
+        stage('Test app with') {
```

Comment:
nit: I think this stage should have a title similar to `Enterprise in Jenkins`, maybe something like `Conjur in K8s Cloud`? `Test app with` feels too nondescript.

Reply:
After looking at how the pipelines displayed graphically - I think this change works. I was evaluating the Jenkinsfile naming without considering it's output.